package.json

```diff
@@ -6,7 +6,7 @@
   "license": "MIT",
   "scripts": {
     "analyze:bundle": "lerna run analyze:bundle --stream",
-    "bootstrap": "lerna bootstrap",
+    "bootstrap": "lerna bootstrap --force-local",
```

Comment:
force local should not be necessary 🤔 . design-system and icons are working together without that specific need